pkg/crc/cluster/kubeadminPassword.go

```diff
@@ -0,0 +1,95 @@
+package cluster
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"math/big"
+
+	"github.com/code-ready/crc/pkg/crc/machine/bundle"
+	"github.com/code-ready/crc/pkg/crc/oc"
+	"golang.org/x/crypto/bcrypt"
+)
+
+// UpdateKubeAdminUserPassword does following
+// - Generate a random password for kubeadmin user
+// - Update the kubeadmin password file
+// - Update the htpasswd secret
+func UpdateKubeAdminUserPassword(kubeAdminPassword string, ocConfig oc.Config, bundle *bundle.CrcBundleInfo) error {
```

Comment:
Is it possible to create a new file with the password in ~/.crc/machines/crc/password ? I don't really like changing a file in the cache after start. It is like "corrupting" it.